public/src/index.js

```diff
@@ -1,37 +1,41 @@
 var LagerApp = React.createClass({
+
   componentDidMount: function() {
     var socket, host;
     host = "ws://localhost:4001";
 
-    function connect() {
+    var connect = function() {
       try {
         socket = new WebSocket(host);
-
         console.log("Socket State: " + socket.readyState);
-
         socket.onopen = function() {
           console.log("Socket Status: " + socket.readyState + " (open)");
         }
-
         socket.onclose = function() {
           console.log("Socket Status: " + socket.readyState + " (closed)");
         }
-
         socket.onmessage = function(msg) {
-          console.log("Received: " + msg.data);
-        }
+          React.findDOMNode(this.refs.line).innerHTML = msg.data;
```

Comment:
Yeah, just a POC. Will be keeping this in a data struct within the React component.